src/api/libcellml/variable.h

```diff
@@ -55,8 +55,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     UnitsPtr getUnits() const;
 
     /**
+     * @brief Set the initial value for this variable using a string.
+     * Set the initial value for this variable using a string.
+     *
+     * @overload
+     * @sa getInitialValue
+     * @param initialValue the initial value to set.
```

Comment:
just for reference, the doxygen is available through the buildbot for each pull request - follow the link to the buildbot results for the pull request, then the documentation builder, and then you should see a link to the "dox". For example, this particular code for this revision is currently available [here](http://autotest.bioeng.auckland.ac.nz/libcellml-buildbot/81db9d9c74d926a0c43d7ba491eb9024c3f9efa4/dox/code/classlibcellml_1_1Variable.html#a4b5f832ae37e025130b8b3880a6a47f8). I think the generated docs get flushed every now and then, so the links are not permanent. #41 is the issue on how these should get integrated into the main project docs on readthedocs.